### ISSUE TITLE
WBO firmware update

### ISSUE THREAD

**Issue by rusefillc** [rusefi/rusefi#5719]
what if one has a board with two WBO modules on board? how is ```setWidebandOffset``` supposed to work if it ```Setting all connected widebands to index %d...```?

what about updateWidebandFirmware? how does one update only one unit?

**Comment by mck1117** [comment]
> one has a board with two WBO modules on board

this is what https://github.com/mck1117/wideband/issues/11 is for, no?

**Comment by rusefillc** [comment]
> 
> this is what [mck1117/wideband#11](https://github.com/mck1117/wideband/issues/11) is for, no?

yes individual units know their own indices thanks to https://github.com/mck1117/wideband/issues/11

but it does not seem like current bootloader custom protocol has provisions to talk only to specified controlled by index as of today? while perfect ideal solution would be to expand protocol it's tempting to just make it work by hanging the other one. I assume that hanging up one of controllers would be way less effort and much less coordination

**Comment by rusefillc** [comment]
@dron0gus now has nissan121 as test hw

![WhatsApp Image 2024-03-05 at 03 50 45_abe6e14e](https://github.com/rusefi/rusefi/assets/48498823/37c6b3c4-eeb1-43ae-9304-eeaa500be86b)
